### PR TITLE
156 storing values in tasks page

### PR DESCRIPTION
**Pull request by mFlaifel**
- solve bug with check box when technical tasks is finished.
- add project URL to the page.

### CODE REVIEW

File: client/src/components/pages/application/Tasks/index.js [hunk 1]
```diff
@@ -55,97 +47,123 @@ const Tasks = () => {
     });
     if (UserId) {
       axios.get(`/api/v1/applicants/${UserId}`).then(({ data: { user } }) => {
-        if (technicalTasks === '') setTechnicalTasks(user.technicalTasks);
+        if (user.technicalTasksLinks) {
+          setTechnicalTasksLinks(user.technicalTasksLinks);
+        }
+        setTechnicalTasks(user.technicalTasks);
```
Comment (Asmaathabet) [inline client/src/components/pages/application/Tasks/index.js:53]: there's problem in technicalTasks value it didn't change to `true` in database when you finish all tasks . I suggest to use Boolean state for it .

File: client/src/components/pages/application/Tasks/index.js [hunk 2]
```diff
@@ -55,97 +47,123 @@ const Tasks = () => {
...
+                      newArr[elementIndex] = !arrayChecks[elementIndex];
+                      setCheckedItem(newArr);
+                      if (!arrayChecks.includes(false)) {
+                        setTechnicalTasks(true);
+                      }
                     }}
+                    checked={
+                      arrayChecks.includes(false) || arrayChecks.length === 0
+                        ? arrayChecks[index]
+                        : true
```
Comment (Asmaathabet) [inline client/src/components/pages/application/Tasks/index.js:117]: checks in check boxes disappear after going to another route and return

File: client/src/utils/application/tasksValidation.js [hunk 3]
```diff
@@ -1,13 +1,13 @@
 import * as yup from 'yup';
 
 const schema = yup.object().shape({
-  TechnicalTasksLinks: yup
+  technicalTasksLinks: yup
     .string()
     .trim()
-    .required()
+    .required('Enter GitHub link')
```
Comment (Asmaathabet) [inline client/src/utils/application/tasksValidation.js:7]:
```suggestion
    .required('Enter your GitHub profile link')
```

Comment (ha77an) [reply client/src/utils/application/tasksValidation.js:7]: we want Project GitHub in here so I can change it to `Enter your GitHub Project link`

Comment (Asmaathabet) [reply client/src/utils/application/tasksValidation.js:7]: yes, exactly

**Comment by Asmaathabet** [comment]
@ha77an 
 please fix the style problems, change position settings , it displays under the drawer

**Pull request merged**